govcd/common_test.go

```diff
@@ -265,15 +265,14 @@ func isTcpPortOpen(host, port string, timeout int) bool {
 
 }
 
-// moved from vapp_test.go
-func createVappForTest(vcd *TestVCD, vappName string) (*VApp, error) {
+// deployVappForTest aims to replace createVappForTest
+func deployVappForTest(vcd *TestVCD, vappName string) (*VApp, error) {
 	// Populate OrgVDCNetwork
-	var networks []*types.OrgVDCNetwork
 	net, err := vcd.vdc.GetOrgVdcNetworkByName(vcd.config.VCD.Network.Net1, false)
 	if err != nil {
 		return nil, fmt.Errorf("error finding network : %s", err)
 	}
-	networks = append(networks, net.OrgVDCNetwork)
+
 	// Populate Catalog
```

Comment:
this and similar comments are not needed, as they are just repeat what is method name

Reply:
We can live with it for now